app/v2/partials/proposal/page.html

```diff
@@ -270,13 +270,22 @@ <h1 class="heading--title">{{proposal.title}}</h1>
           <h2 class="heading--header">
             <i class="fa fa-file-text"></i>&nbsp;&nbsp; {{'Proposal Text' | translate }}
           </h2>
-        </div>
-        <div class="etherpad__wrapper">
+        </div>    
+
+        <div ng-if="!campaignConfigs['appcivist.campaign.disable-etherpad']" class="etherpad__wrapper">
           <iframe ng-if="!userIsAuthor" name="embed_readonly" ng-src="{{etherpadReadOnlyUrl}}" width="100%" height="600px">
           </iframe>
           <iframe ng-if="userIsAuthor" name="embed_readwrite" ng-src="{{etherpadReadWriteUrl}}" width="100%" height="600px">
           </iframe>
         </div>
+        
+        <div ng-if="campaignConfigs['appcivist.campaign.disable-etherpad']">
```

Comment:
should be `ng-if="campaignConfigs['appcivist.campaign.disable-etherpad']==='TRUE'"`
Otherwise, not having the configuration (which would make the variable `undefined` or `null`) would also evaluate as `TRUE`